typescript/phoenix-sdk/src/market.ts

```diff
@@ -113,9 +113,14 @@ export type L3UiBook = {
   asks: L3UiOrder[];
 };
 
+export type UiLadderLevel = {
+  price: number;
+  quantity: number;
```

Comment:
i think we use priceAsFloat and sizeInRawBaseUnits elsewhere

Reply:
Gi